### ISSUE TITLE
Automatically Populate from Config

### ISSUE THREAD

**Issue by aliebuck** [cykoder/wp2anchor#4]
Assuming import.php is in the root of the Anchor website it would be convenient to just grab the database config from `/anchor/config/db.php`.

**Comment by cykoder** [comment]
That's true, I haven't worked on this in a while - I'll fix this issue when I get some free time later today
